src/span_context.js

```diff
@@ -26,6 +26,7 @@ export default class SpanContext {
   _baggage: any;
   _debugId: ?string;
   _samplingState: SamplingState;
+  _remote: boolean; // indicates that span context represents a remote parent
```

Comment:
Replace with `isRemoteParent`?

Reply:
In Java Beans naming we don't add `is` to the field names. And `parent` would be incorrect here because it's _this span context_ that is "remote".